kubos/versions.py

```diff
@@ -24,7 +24,7 @@
 from kubos.utils.constants import *
 
 def addOptions(parser):
-    pass
+    parser.add_argument('-a', '--all-versions', action='store_false', default=True, help='Show all available minor versions')
```

Comment:
Should the default be False? I feel like the average user probably doesn't care about seeing all of our intermediate builds